PF/Assignment4/TowerOfHanoiTest.java

```diff
@@ -0,0 +1,56 @@
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class TowerOfHanoiTest {
+
+	@BeforeClass
+	public static void setUpBeforeClass() throws Exception {
+	}
+
+	@AfterClass
+	public static void tearDownAfterClass() throws Exception {
+	}
+
+	@Before
+	public void setUp() throws Exception {
+	}
+
+	@After
+	public void tearDown() throws Exception {
+	}
+
+	@Test
+	public void test() {
```

Comment:
Tests are good. But from next time on wards create different test methods for each test case. For example if u want to test 3 cases these should be done in 3 different methods (one in each). In this way it is easy to identify test cases.